src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingNamedTypeSymbol.cs

```diff
@@ -273,6 +274,23 @@ private static ExtendedErrorTypeSymbol CyclicInheritanceError(RetargetingNamedTy
             return new ExtendedErrorTypeSymbol(declaredBase, LookupResultKind.NotReferencable, info, true);
         }
 
+        internal override TypeSymbol ExtensionClassTypeNoUseSiteDiagnostics
+        {
+            get
+            {
+                if (ReferenceEquals(_lazyExtendedType, ErrorTypeSymbol.UnknownResultType))
+                {
+                    var underlyingExtendedType = _underlyingType.ExtensionClassTypeNoUseSiteDiagnostics;
+                    if ((object)underlyingExtendedType != null)
+                    {
+                        underlyingExtendedType = this.RetargetingTranslator.Retarget(underlyingExtendedType, RetargetOptions.RetargetPrimitiveTypesByName);
```

Comment:
I don't believe so, no - I've recorded that that needs to be done, though (there are other higher-priority things that need to be done first).